java/client/src/main/java/glide/api/models/commands/InfoOptions.java

```diff
@@ -0,0 +1,62 @@
+package glide.api.models.commands;
+
+import glide.api.commands.ServerCommands;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import lombok.Builder;
+import lombok.Singular;
+
+/** Object builder to add optional arguments to {@link ServerCommands#info(InfoOptions)} */
+@Builder
```

Comment:
Add a code example here

Reply:
Separate PR.  Will need to udpate python and node too.